src/transformers/convert_slow_tokenizer.py

```diff
@@ -1050,6 +1051,41 @@ def post_processor(self):
         )
 
 
+class SiglipConverter(SpmConverter):
+    def normalizer(self, proto):
+        precompiled_charsmap = proto.normalizer_spec.precompiled_charsmap
+
+        list_normalizers = []
+
+        if self.original_tokenizer.do_lower_case:
+            list_normalizers.append(normalizers.Lowercase())
+            list_normalizers.extend([normalizers.Replace(i, "") for i in string.punctuation])
```

Comment:
Regex is gonna be faster for all punctuation. Format the regex earlier with the string

Reply:
What's the syntax here? I tried this:
```
list_normalizers.extend([normalizers.Replace(re.escape(string.punctuation), "")])
```